addons/website_event_track/static/src/js/website_event_track_proposal_form.js

```diff
@@ -0,0 +1,173 @@
+    'input input[name="partner_name"]': '_onPartnerNameInput',
+    'click .o_wetrack_proposal_submit_button': '_onProposalFormSubmit',
+},
+
+/**
+ * @override
+ */
+init: function () {
+    this._super(...arguments);
+    this.useAdvancedContact = false;
+},
+
+//--------------------------------------------------------------------------
+// Private
+//--------------------------------------------------------------------------
+
+/**
+ * Evaluate and return validity of form input fields: 
+ * - 1) error 'invalidFormInputs' : Invalid ones are marked as is-invalid and o_wetrack_input_error.
+ * - 2) error 'noContactMean' : Contact mean fields are marked as is-invalid and contact
+ * section as o_wetrack_no_contact_mean_error if none of them is filled.
+ *  
+ * @private
+ * @returns {Boolean} - True if no error remain, false otherwise
+ */
+_isFormValid: function () {
+    var formErrors = [];
+
+    // 1) Valid Form Inputs
+    this.$('.form-group').each(function (index, field) { 
+        var $field = $(field);
+        // Validate current input, if not select2 field.
+        var inputs = $field.find('.form-control').not('.o_wetrack_select2_tags');
+        var invalidInputs = inputs.toArray().filter(function (input) {
+            return !input.checkValidity();
+        });
+        
+        $field.find('.form-control').removeClass('o_wetrack_input_error is-invalid');
+        if (invalidInputs.length) {
+            $field.find('.form-control').addClass('o_wetrack_input_error is-invalid');
+            formErrors.push('invalidFormInputs');
+        }
+    });
+
+    // 2) Advanced Contact Must Have a Contact Mean
+    if (this.useAdvancedContact) {
+        var hasContactMean = this.$('.o_wetrack_contact_phone_input').val() || 
+                                this.$('.o_wetrack_contact_email_input').val();
+        if (!hasContactMean) { 
+            this.$('.o_wetrack_contact_information').addClass('o_wetrack_no_contact_mean_error'); 
+            this.$('.o_wetrack_contact_mean').addClass('is-invalid');
+            formErrors.push('noContactMean');
+        } else {
+            this.$('.o_wetrack_contact_information').removeClass('o_wetrack_no_contact_mean_error'); 
+            this.$('.o_wetrack_contact_mean:not(".o_wetrack_input_error")').removeClass('is-invalid');
+        }
+    }
+
+    // Form Validity and Error Display
+    this._updateErrorDisplay(formErrors);
+    return formErrors.length === 0;
+},
+
+/**
+ * If there are still errors in form, display the error section and
+ * compose the error message accordingly.
+ * 
+ * @private
+ * @param {Array} errors - Names of errors still present in form.
+ */
+_updateErrorDisplay: function(errors) {
+
+    if (!errors.length) {
+        this.$('.o_wetrack_proposal_error_section').addClass('d-none');
+    } else { 
+        this.$('.o_wetrack_proposal_error_section').removeClass('d-none');
+    }
+
```

Comment:
this.$('.o_wetrack_proposal_error_section').toggleClass('d-none', !errors.length);